sdk/go/azure/webpubsub/customCertificate.go

```diff
@@ -14,6 +14,123 @@ import (
 
 // Manages an Azure Web PubSub Custom Certificate.
 //
+// ## Example Usage
+//
+// ```go
+// package main
+//
+// import (
+//
+//	"github.com/pulumi/pulumi-azure/sdk/v5/go/azure/core"
+//	"github.com/pulumi/pulumi-azure/sdk/v5/go/azure/keyvault"
+//	"github.com/pulumi/pulumi-azure/sdk/v5/go/azure/webpubsub"
+//	"github.com/pulumi/pulumi-azurerm/sdk/v1/go/azurerm"
```

Comment:
Are we importing a provider that doesn't exist. `github.com/pulumi/pulumi-azurerm` redirects to `github.com/pulumi/pulumi-azure-native`, do we mean that?

Reply:
https://github.com/pulumi/pulumi-terraform-bridge/issues/1712